File: src/jp/senchan/android/wasatter/StatusItemComparator.java
/**
 *
 */
package jp.senchan.android.wasatter;

import java.util.Comparator;

/**
 * @author Senka/Takuji
 * 
 */
public class StatusItemComparator implements Comparator<WasatterItem> {
    @Override
    public int compare(WasatterItem object1, WasatterItem object2) {
        // TODO 自動生成されたメソッド・スタブ
        return object1.epoch > object2.epoch ? -1 : 1;
    }

}

File: app/src/main/java/jp/senchan/android/wasatter/StatusItemComparator.kt
/**
 *
 */
package jp.senchan.android.wasatter

import java.util.*

/**
 * @author Senka/Takuji
 */
class StatusItemComparator : Comparator<WasatterItem> {
    override fun compare(object1: WasatterItem, object2: WasatterItem): Int { // TODO 自動生成されたメソッド・スタブ
        return if (object1.epoch > object2.epoch) -1 else 1
    }
}